exercises/ruby-1-b/README.md

```diff
@@ -4,17 +4,27 @@ This is Part 1 of our research into how people write Ruby differently. This exer
 
 ## Instructions
```

Comment:
This README needs to be a full spec of what needs to happen. People cannot see the tests when solving this exercise. So things like `.published` and the other methods with their meanings and things like `ArgumentError` all need explicitly specifying in here.